content/applications/marketing/email_marketing.rst

```diff
@@ -1,440 +1,792 @@
+specified recipient filter, in an equation-like format, which appears beneath the
+:guilabel:`Recipients` field.
 
-The :guilabel:`Recipients` field options are as follows:
+The :guilabel:`Recipients` field options, other than the default :guilabel:`Mailing List` option,
+are as follows:
 
-- :guilabel:`Applicant`: filter focuses on specific job applicants in the database.
-- :guilabel:`Contact`: filter focuses on specific contacts in the database.
-- :guilabel:`Event Registration`: filter focuses on people in the database who purchased event
+- :guilabel:`Contact`: focuses on specific contacts in the database.
+- :guilabel:`Event Registration`: focuses on people in the database who purchased event
   registrations.
-- :guilabel:`Event Track`: filter focuses on people in the database who hosted a specific talk
-  (track) at an event.
-- :guilabel:`Lead/Opportunity`: filter focuses on leads or opportunities in the database.
-- :guilabel:`Mailing Contact`: filter focuses on specific mailing contacts in the database.
-- :guilabel:`Sales Order`: filter focuses on a specific sales orders in the database.
+- :guilabel:`Lead/Opportunity`: focuses on leads or opportunities in the database.
+- :guilabel:`Mailing Contact`: focuses on specific mailing contacts in the database.
+- :guilabel:`Sales Order`: focuses on a specific sales orders in the database.
 
-If the specified recipient fields don't automatically reveal themselves, simply click the
-:guilabel:`Add Filter` button beneath the :guilabel:`Recipients` field, and Odoo reveals the
-necessary equation fields to further drill down the target recipients for this mailing.
+Add recipient filter
+~~~~~~~~~~~~~~~~~~~~
 
-Add a recipient filter
-----------------------
+To add a more specific recipient filter to any :guilabel:`Recipient` option, select any recipient
+option (other than :guilabel:`Mailing List`), and click the :guilabel:`Modify filter (right-facing
+arrow)` icon beneath the :guilabel:`Recipient` field to reveal three subsequent filter rule fields,
+formatted like an equation.
 
-To add a more specified recipient filter, select any recipient option (other than :guilabel:`Mailing
-List`), and clicking :guilabel:`Add Filter`, if needed, to reveal three fields, formatted like an
-equation.
+To reveal the sub-menu options within the filter rule fields, click each field, and make the desired
+selections, until the preferred configuration has been achieved.
 
-To reveal the sub-menu options, click each field, and make the desired selections, until the
-preferred configuration has been achieved. The number of :guilabel:`Records` that match the rule(s)
-are indicated to the right of the :guilabel:`Recipients` field, in green.
+The number of :guilabel:`records` in the database that match the configured rule(s) are indicated
+beneath the configured filter rule(s), in green.
 
-.. image:: email_marketing/add-filter-button.png
+.. image:: email_marketing/filter-records.png
```

Comment:
Similar to the comment above. This is not a great example of how to use filters in a real-world setting.

Reply:
Since I added the above screenshot in an example admonition, I think this can still serve a purpose -- not as a 'real world' setting example, but more of how the rule lines appear on the U/I, and how the 'Records' look on screen, as well. However, if you feel like this screenshot still isn't useful, just let me know. Thanks.